### PR TITLE
Add PE image support in scripts

### PR DESCRIPTION
**Pull request by catalindumitru**
Created a .config file for aarch64 and added the `pe` option throughout the scripts.

### CODE REVIEW

File: tests/tools/s3/resource_manifest.json [hunk 1]
```diff
@@ -51,5 +51,132 @@
...
+      "image_format": "elf",
+      "halt_after_boot": false,
+      "distro": "Ubuntu"
+    },
+    {
+      "resource_type": "disk",
+      "resource_name": "ubuntu-focal-rootfs.ext4",
+      "relative_path": "disk/",
+      "distro": "Ubuntu",
+      "compatible_kernels": "v1/kernel/bzimage/bzimage-focal|v1/kernel/elf/vmlinux-focal"
```
Comment (andreeaflorescu) [inline tests/tools/s3/resource_manifest.json:179]: Can you also add the following intird configuration:
```
    {
      "resource_type": "initrd",
      "resource_name": "aarch64-initramfs.cpio",
      "relative_path": "kernel/pe"
    }
```

Comment (catalindumitru) [reply tests/tools/s3/resource_manifest.json:179]: Added it.

Comment (lauralt) [reply tests/tools/s3/resource_manifest.json:179]: Should we add it to v2 as well?

**Comment by lauralt** [comment]
We should also mention about the kernel version update in the first commit message.

File: tests/tools/s3/resource_manifest.json (hunk 1 above)
Comment (andreeaflorescu) [reply tests/tools/s3/resource_manifest.json:179]: I think we shouldn't. I'll remove it from the v2 directory in S3, that was a mistake on my part, sorry.

**Review by lauralt — DISMISSED** [review]

**Review by andreeaflorescu — APPROVED** [review]

**Review by lauralt — DISMISSED** [review]

File: tests/tools/s3/resource_manifest.json [hunk 2]
```diff
@@ -51,5 +51,137 @@
...
```
Comment (lauralt) [inline tests/tools/s3/resource_manifest.json:112]: It is compatible with v2 ones as well, so we could probably update this; same comment for v3.

**Review by andreeaflorescu — APPROVED** [review]

**Review by lauralt — APPROVED** [review]

**Pull request merged**